docs/networkpolicy-recommendation.md

```diff
@@ -0,0 +1,148 @@
+  - [Retrieve the result of a policy recommendation job](#retrieve-the-result-of-a-policy-recommendation-job)
+<!-- /toc -->
+
+## Introduction
+
+Theia NetworkPolicy Recommendation recommends the NetworkPolicy configuration to secure Kubernetes
+network and applications. It analyzes the network flows collected by
+[Grafana Flow Collector](network-flow-visibility.md#grafana-flow-collector) to generate
+[Kubernetes NetworkPolicies](https://kubernetes.io/docs/concepts/services-networking/network-policies/)
+or [Antrea NetworkPolicies](https://github.com/antrea-io/antrea/blob/main/docs/antrea-network-policy.md).
+This feature assists cluster administrators and app developers in securing their applications
+according to Zero Trust principles.
+
+## Installation
+
+Theia's NetworkPolicy Recommendation capablity leverages Antrea Flow Exporter and Flow Aggreator.
+Before leveraging this feature, follow this [doc](https://github.com/antrea-io/antrea/blob/main/docs/network-flow-visibility.md)
+to ensure they are deployed. In addition, ensure `podLabels: true` is set in [Flow Aggregator Configuration](https://github.com/antrea-io/antrea/blob/main/docs/network-flow-visibility.md#configuration-1),
+as Theia's NetworkPolicy Recommendation feature needs to process pod label information.
+
+Then, please follow these [deployment steps](network-flow-visibility.md#deployment-steps)
+to deploy Theia.
+
+Commet out following lines since user won't need to deploy Spark Operator separately after PR#31 merged.
+
+<!-- After we finish the deployment of Grafana Flow Collector and can see some flow records in the Grafana UI,
+we can continue to deploy other necessary components of policy recommendation feature:
+
+We need to install the [Kubernetes Operator for Apache Spark](https://github.com/GoogleCloudPlatform/spark-on-k8s-operator)
+in the cluster. Our policy recommendation logic is implemented as a [Spark](https://github.com/apache/spark)
+application, the Kubernetes Operator for Apache Spark helps us schedule the Spark job in the Kubernetes cluster.
+
+Please run the following commands:
+
+```bash
+helm repo add spark-operator https://googlecloudplatform.github.io/spark-on-k8s-operator
+helm install policy-reco spark-operator/spark-operator --namespace flow-visibility --set image.tag=latest
+```
+
+This will install the Kubernetes Operator for Apache Spark into the `flow-visibility` Namespace.
+
+Once we finish these deployment steps, we should see the `policy-reco-spark-operator` Pod running inside the `flow-visibility` namespace:
+
+```bash
+$ kubectl get pods -n flow-visibility
+NAME                                          READY   STATUS    RESTARTS   AGE
+policy-reco-spark-operator-56c4cb454c-4vhfh   1/1     Running   0          2m19s
+``` -->
+
+## Perform NetworkPolicy Recommendation
+
+Currently, the NetworkPolicy recommendation feature only supports command line interaction through `theia`.
```

Comment:
Opinion: I would rephrase this as "Users can leverage Theia's NetworkPolicy recommendation feature through `theia` CLI."